Tests/BasicsTests/SQLiteBackedCacheTests.swift

```diff
@@ -55,8 +55,11 @@ final class SQLiteBackedCacheTests: XCTestCase {
     }
 
     func testFileDeleted() throws {
+#if os(Windows)
+        throw XCTSkip("open file cannot be deleted on Windows")
+#else
```

Comment:
why do we need the else? does the code not compile windows?

Reply:
Avoids a warning - "code after throw will never be executed"

Reply:
we have allot of tests with the following:

```
#if !os(macOS)
try XCTSkipIf(true, "test is only supported on macOS")
#endif
```

does that help?

Reply:
Sure, that seems to avoid the warning, as long as we can agree to disagree (I personally find `XCTSkipIf(true, ...` terrible).